config/config.go

```diff
@@ -174,6 +174,7 @@ func CreateApp(v *viper.Viper) (_ *App, err error) {
 	}
 
 	sources := createSources(v)
+	sources["stdout"] = v1.NewSource(&utils.IoSourceFactory{}, &utils.IoSourceFactory{}, &viper.Viper{}, "stdout")
```

Comment:
Leftovers from testing?

Reply:
Thought of adding it as a useful feature, but it can be removed